npm/src/upload/upload-form.ts

```diff
@@ -235,6 +237,18 @@ export class UploadForm {
     return parentFolder
   }
 
+  private showUploadingRecordsPage() {
+    const fileUpload: HTMLDivElement | null =
+      document.querySelector("#file-upload")
+    const progressBar: HTMLDivElement | null =
+      document.querySelector("#progress-bar")
+
+    if (fileUpload && progressBar) {
```

Comment:
Does this new step mean that the code in the progress callback can be removed? https://github.com/nationalarchives/tdr-transfer-frontend/blob/ea95579297bc22640b548d3b27e227722932aa1a/npm/src/clientfileprocessing/index.ts#L36

Reply:
🤦‍♂️  ah, yes. I had a feeling I was missing something.